tests/Resolver/ResolveAliasesTest.php

```diff
@@ -31,63 +31,63 @@ public function testResolveAliases(array $expectedAnswers, array $answers, $name
                     new Record('igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.133'),
-                ),
+                ],
                 'igor.io',
-            ),
-            array(
-                array('178.79.169.131'),
-                array(
+            ],
+            [
+                ['178.79.169.131'],
+                [
                     new Record('igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
                     new Record('foo.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
                     new Record('bar.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
-                ),
+                ],
                 'igor.io',
-            ),
-            array(
-                array('178.79.169.131'),
-                array(
+            ],
+            [
+                ['178.79.169.131'],
+                [
                     new Record('igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'foo.igor.io'),
                     new Record('foo.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
-                ),
+                ],
                 'igor.io',
-            ),
-            array(
-                array('178.79.169.131'),
-                array(
+            ],
+            [
+                ['178.79.169.131'],
+                [
                     new Record('igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'foo.igor.io'),
                     new Record('foo.igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'bar.igor.io'),
                     new Record('bar.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
-                ),
+                ],
                 'igor.io',
-            ),
-            array(
-                array('178.79.169.131', '178.79.169.132', '178.79.169.133'),
-                array(
+            ],
+            [
+                ['178.79.169.131', '178.79.169.132', '178.79.169.133'],
+                [
                     new Record('igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'foo.igor.io'),
                     new Record('foo.igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'bar.igor.io'),
                     new Record('bar.igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'baz.igor.io'),
                     new Record('bar.igor.io', Message::TYPE_CNAME, Message::CLASS_IN, 3600, 'qux.igor.io'),
                     new Record('baz.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.131'),
                     new Record('baz.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.132'),
                     new Record('qux.igor.io', Message::TYPE_A, Message::CLASS_IN, 3600, '178.79.169.133'),
-                ),
+                ],
                 'igor.io',
-            ),
-        );
+            ],
+        ];
```

Comment:
Same as `ParserTest`, sadly GitHub won't let me suggest the changes due to deleted lines.

Reply:
Good catch, adjusted as suggested :+1: